vars/mavenBuildSnapshot.groovy

```diff
@@ -15,7 +15,7 @@ def call(body) {
         // update any versions that we want to override
         overwriteDeps(config.pomVersionToUpdate)
 
-        sh "mvn clean -e -U install -Dm2.repo=.mvnrepository -Duser.home=/root/ -Ddocker.push.registry=${env.FABRIC8_DOCKER_REGISTRY_SERVICE_HOST}:${env.FABRIC8_DOCKER_REGISTRY_SERVICE_PORT}"
+        sh "mvn clean -e -U install -Ddocker.push.registry=${env.FABRIC8_DOCKER_REGISTRY_SERVICE_HOST}:${env.FABRIC8_DOCKER_REGISTRY_SERVICE_PORT}"
```

Comment:
@jaseemabid I think we need to add `Dm2.repo`, `Duser.home` switches back for respective location